scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1735,7 +1735,7 @@ class Router(formatOps: FormatOps) {
           Split(Space, 0, policy = policy)
             .withIndent(if (bodyBlock) 0 else 2, expire, After)
             .withIndent(
-              if (bodyBlock) 4 else style.continuationIndent.caseSite,
+              style.continuationIndent.caseSite + (if (bodyBlock) 0 else -2),
```

Comment:
please add extensive tests with values 0 and 1.

Reply:
Added